### PR TITLE
Prompt to use module when given plain module name

### PR DESCRIPTION
**Pull request by acammack-r7**
During the mini Metasploitable3 CTF, I noticed a lot of people who had trouble getting a module loaded. I have added a piece of fallback code for when just a module name is given on the command line that will prompt the user to see if they wanted to load that module.

## Verification

- [x] Start `msfconsole`
- [x] `exploit/windows/smb/ms08_067_netapi`
- [x] Get prompted
- [x] **Verify** that `y` (and equivalent) loads the module
- [x] **Verify** anything else does not load the module and does not mess up your prompt

### CODE REVIEW

File: lib/msf/ui/console/driver.rb [hunk 1]
```diff
@@ -504,6 +504,15 @@ def unknown_command(method, line)
         end
         self.busy = false
         return
+      elsif framework.modules.create(method)
+        super
+        print_error("This is a module we can load. Do you want to use #{method}?")
```
Comment (wvu-r7) [inline lib/msf/ui/console/driver.rb]: Looks like `prompt_yesno` isn't available in this context. Could you change this interaction to match that? I'd like the user experience to be consistent.

Also, if you interrupt the input line, the prompt doesn't get restored right away:

```
msf5 > exploit/windows/smb/ms17_010_eternalblue
[-] Unknown command: exploit/windows/smb/ms17_010_eternalblue.
[-] This is a module we can load. Do you want to use exploit/windows/smb/ms17_010_eternalblue?
(y/N) > Interrupt: use the 'exit' command to quit
(y/N) > N
[-] Unknown command: N.
(y/N) > y
[-] Unknown command: y.
(y/N) > exploit/windows/smb/ms17_010_eternalblue
[-] Unknown command: exploit/windows/smb/ms17_010_eternalblue.
[-] This is a module we can load. Do you want to use exploit/windows/smb/ms17_010_eternalblue?
(y/N) > y
msf5 exploit(windows/smb/ms17_010_eternalblue) > back
msf5 >
```

Sadly, due to how we implement prompts, it's kinda fair game what gets to change it.

Comment (acammack-r7) [reply lib/msf/ui/console/driver.rb]: Ah, all good points @wvu-r7. Fixed, and refactored into a method in `shell.rb` for future use due to the gotchas.

**Comment by wvu-r7** [comment]
```
msf5 > exploit/windoze/smb/ms08_067_netapi
[-] Unknown command: exploit/windoze/smb/ms08_067_netapi.
msf5 > exploit/windows/smb/ms08_067_netapi
[-] Unknown command: exploit/windows/smb/ms08_067_netapi.
This is a module we can load. Do you want to use exploit/windows/smb/ms08_067_netapi? [y/N]  Interrupt: use the 'exit' command to quit
msf5 > exploit/windows/smb/ms08_067_netapi
[-] Unknown command: exploit/windows/smb/ms08_067_netapi.
This is a module we can load. Do you want to use exploit/windows/smb/ms08_067_netapi? [y/N]
msf5 > exploit/windows/smb/ms08_067_netapi
[-] Unknown command: exploit/windows/smb/ms08_067_netapi.
This is a module we can load. Do you want to use exploit/windows/smb/ms08_067_netapi? [y/N]  N
msf5 > exploit/windows/smb/ms08_067_netapi
[-] Unknown command: exploit/windows/smb/ms08_067_netapi.
This is a module we can load. Do you want to use exploit/windows/smb/ms08_067_netapi? [y/N]  y
msf5 exploit(windows/smb/ms08_067_netapi) > back
msf5 >
```

:tada:

**Comment by wvu-r7** [comment]
Very nice. This is a great usability improvement. Thanks!

**Comment by wvu-r7** [comment]
# Release Notes

`msfconsole` will now prompt the user to load a module if they specify its name without the `use` command.

**Pull request merged**